src/tcgwars/logic/util/PokemonCardSet.java

```diff
@@ -127,6 +133,14 @@ public PokemonCard getTopNonBreakPokemonCard() {
     return null; //Should an IllegalStateException be thrown here, same as above?
   }
 
+  public PokemonCard getTopNonLevelUpPokemonCard() {
+    for (Card card : cards().filterByType(CardType.POKEMON)) {
+      if (card.getCardTypes().is(CardType.LVL_X)) continue;
```

Comment:
Just to avoid issues with Ditto (Fossil), I'd look for static cardTypes here, similar to the change for top non BREAK card in #933

```suggestion
      if (card.getStaticCardTypes().is(CardType.LVL_X)) continue;
```